xata/namespace.py

```diff
@@ -70,6 +72,10 @@ def request(self, http_method: str, url_path: str, headers: dict = {}, payload:
         else:
             resp = request(http_method, url, headers=headers, json=payload)
 
+        # log server message
+        if "x-xata-message" in resp.headers:
+            self.logger.warn(resp.headers["x-xata-message"])
```

Comment:
Does it make sense to have multiple log levels with different header names?

X-Xata-Error
X-Xata-Warn
X-Xata-Info

Reply:
Adding levels does make sense but in a different format. For now, the server only emits warnings, as soon as we need granularity in the severity level, we'll add a prefix to the string. E.g. for a warning we would prefix it with `warning#` -> `warning#this is a warning message`.